<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="bowlAV">
    <link name="link">
      <inertial>
      <pose>0 0 0.0175 0 0 0</pose>
        <mass>0.1</mass>
	  <inertia>
	    <ixx>0.000250308</ixx>
	    <ixy>0.0</ixy>
	    <ixz>0.0</ixz>
	    <iyy>0.000250308</iyy>
	    <iyz>0.0</iyz>
	    <izz>0.0004802</izz>
	  </inertia>
      </inertial>
      <collision name="collision">
        <pose>0 0 0.0175 0 0 0</pose>
        <geometry>
          <cylinder>
            <radius>0.098</radius>
            <length>0.035</length>
          </cylinder>
        </geometry>
        <surface>
          <contact>
            <ode>
              <max_vel>0.1</max_vel>
              <min_depth>0.001</min_depth>
            </ode>
          </contact>
          <friction>
            <ode>
              <mu>1.0</mu>
              <mu2>1.0</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://bowlAV/meshes/bowl.dae</uri>
          </mesh>
        </geometry>
        <material>
          <ambient>1 0 0 1</ambient>
          <diffuse>1 0 0 1</diffuse>
          <specular>0.1 0.1 0.1 1</specular>
          <emissive>0 0 0 0</emissive>
        </material>
      </visual>
    </link>
  </model>
</sdf>
